src/interfaces/swig/GaussianProcess.i

```diff
@@ -6,102 +6,23 @@
 
 /* Remove C Prefix */
 %shared_ptr(shogun::MeanFunction)
-%shared_ptr(shogun::ZeroMean)
```

Comment:
did you grep for any use of these classes anywhere in the source tree? Just to avoid stale API stuff lingering somewhere

Reply:
will do :)